music/src/transcription/transcription_test.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * @license
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as tf from '@tensorflow/tfjs-core';
+import * as test from 'tape';
+
+import {NoteSequence} from '../protobuf';
+import {FRAME_LENGTH_SECONDS, MIDI_PITCHES, MIN_MIDI_PITCH, pianorollToNoteSequence} from './model';
+
+test('PianorollToNoteSequence', (t: test.Test) => {
+  const frames = tf.buffer([300, MIDI_PITCHES]);
+  const onsets = tf.buffer([300, MIDI_PITCHES]);
+  // Activate key 39 for the middle 50 frames and last 10 frames.
+  for (let i = 25; i < 75; ++i) {
+    frames.set(0.6, i, 39);
```

Comment:
nit: i would put the 0.6 in a constant -- it comes up a lot and i wouldn't know what it means immediately otherwise

Reply:
Done